### PR TITLE
2023-04-28 Jonathan Pallant

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

File: _episodes/interviews/2023-04-28-jonathan-pallant.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Timestamps
+
+- [@0:00] - Introduction
+- [@2:23] - Jonathan's projects
+- [@10:3] - Keyboard drivers, CPM, IBM history
+- [@22:19] - Jonathan's background in embedded systems
+- [@23:46] - Understanding more about computers and laptops
+- [@33:37] - Rust working group for embedded systems
+- [@37:16] - The power of rust on microcontrollers
```
Comment (thejpster) [inline _episodes/interviews/2023-04-28-jonathan-pallant.md:29]: Rust, capital R

File: _episodes/interviews/2023-04-28-jonathan-pallant.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+- [@0:00] - Introduction
+- [@2:23] - Jonathan's projects
+- [@10:3] - Keyboard drivers, CPM, IBM history
```
Comment (thejpster) [inline _episodes/interviews/2023-04-28-jonathan-pallant.md:25]: CP/M

**Review by thejpster — APPROVED** [review]
Excited to listen back to this one! Spotted a couple of typos.

**Comment by thejpster** [comment]
Listening through, there are a couple of instances where the audio is duplicated. This is perhaps where someone came in and interrupted us and I repeated myself. Is it useful if I have another go-through and note where these are so they can be trimmed out, or shall we just leave it?

**Comment by allenwyma** [comment]
It’s a bit late to be making edits but can try. Do you know where?

**Comment by thejpster** [comment]
I didn't take notes, so I'd have to listen to it all again from start to finish.

File: _episodes/interviews/2023-04-28-jonathan-pallant.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- [@0:00] - Introduction
+- [@2:23] - Jonathan's projects
+- [@10:3] - Keyboard drivers, CP/M, IBM history
+- [@22:19] - Jonathan's background in embedded systems
+- [@23:46] - Understanding more about computers and laptops
+- [@33:37] - Rust working group for embedded systems
+- [@37:16] - The power of Rust on microcontrollers
+- [@40:00] - The difference between a Microcontroller and a Central Processing Unit
+- [@42:15] - Discussing System on Chip (SoC), Memory Management Unit (MMU) & Microcontrollers
```
Comment (jonhoo) [inline _episodes/interviews/2023-04-28-jonathan-pallant.md:31]:
```suggestion
- [@42:15] - Discussing System on Chip (SoC), Memory Management Unit (MMU), and Microcontrollers
```

File: _episodes/interviews/2023-04-28-jonathan-pallant.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- [@45:50] - RISC (Reduced Instruction Set Computing) vs CISC (Complex Instruction Set Computing) architecture.
```
Comment (jonhoo) [inline _episodes/interviews/2023-04-28-jonathan-pallant.md:32]:
```suggestion
- [@45:50] - RISC (Reduced Instruction Set Computing) vs CISC (Complex Instruction Set Computing) architecture
```

File: _episodes/interviews/2023-04-28-jonathan-pallant.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- [@53:30] - How did the working group become an official working group?
+- [@56:00] - Using the Rust programming language for working with microcontrollers and developing systems.
```
Comment (jonhoo) [inline _episodes/interviews/2023-04-28-jonathan-pallant.md:34]:
```suggestion
- [@56:00] - Using Rust with microcontrollers
```

File: _episodes/interviews/2023-04-28-jonathan-pallant.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+- [@59:54] - Why Rust is suitable for most projects versus C or Python
```
Comment (jonhoo) [inline _episodes/interviews/2023-04-28-jonathan-pallant.md:35]:
```suggestion
- [@59:54] - Choosing Rust over C or Python
```

**Review by jonhoo — APPROVED** [review]

File: _episodes/interviews/2023-04-28-jonathan-pallant.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Rust Embedded WG with Jonathan Pallant"
+date: 2023-04-28T15:46:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-04-28-jonathan-pallant.md:3]:
```suggestion
date: 2023-04-28T16:25:00Z
```

**Pull request merged**